src/freenas/usr/local/bin/ixdiagnose

```diff
@@ -55,7 +59,8 @@ dumpfile_to_submit_exists ()
 truncate_files ()
 {
 	local limit=$1
-	local IFS=$'\n'
+	local IFS='
```

Comment:
Can you please elaborate?

Reply:
We have different quotes behaviour in linux where value of `ixdiagnose` changes to `ixdiag ose` with n being stripped out. This keeps the IFS set to newline and is consistent with both platforms.